libs/compute_api/src/spec.rs

```diff
@@ -183,6 +189,8 @@ pub enum ComputeFeature {
     /// track short-lived connections as user activity.
     ActivityMonitorExperimental,
 
+    PrewarmLfcOnStartup,
```

Comment:
I think it shouldn't be a part of feature flags, which are meant to be temporary. Instead, it should be a part of the main spec body because it's a permanent feature/mode

(This `ActivityMonitorExperimental` is a bit special, I left it for longer, because statistics-based monitoring is tricky, but we should also remove it already)